couchpotato/core/notifications/emby.py

```diff
@@ -0,0 +1,89 @@
+import json
+import urllib, urllib2
+
+from couchpotato.core.helpers.variable import cleanHost
+from couchpotato.core.logger import CPLog
+from couchpotato.core.notifications.base import Notification
+
+
+log = CPLog(__name__)
+
+autoload = 'Emby'
+
+
+class Emby(Notification):
+
+    def notify(self, message = '', data = None, listener = None):
+        host = self.conf('host')
+        apikey = self.conf('apikey')
+
+        host = cleanHost(host)
+        url = '%semby/Library/Series/Updated' % (host)
```

Comment:
Added makes more sense in this situation. I guess we could call Updated when Couchpotato downloads a new quality, but I'm not sure if we would know that going into this script.